#!/bin/sh /usr/share/dpatch/dpatch-run
Description: gcc4.5 and later are more restrictive with c++ function calls,
 this trivial patch fixes build failures with -fpermissive
Author: Andreas Moog <amoog@ubuntu.com>
Last-Update: 2011-07-15

=== modified file 'src/njamedit.cpp'
--- a/src/njamedit.cpp	2006-07-27 23:15:50 +0000
+++ b/src/njamedit.cpp	2011-07-15 20:01:54 +0000
@@ -114,7 +114,7 @@
 				key = SDLK_a;
 			else if (CheckForSave())
 			{
-				LogFile::LogFile("Saving maps");
+				LogFile("Saving maps");
 				m_Maps.Save(filename);
 				level_type_was = level_type;
 				changed = false;
@@ -131,7 +131,7 @@
 					char types[2][5] = { "COOP", "DUEL" };
 					sprintf(buf, "levels/%s.%s\0", filename, types[level_type]);
 					sprintf(filename, "%s\0", buf);
-					LogFile::LogFile("Saving maps");
+					LogFile("Saving maps");
 					m_Maps.Save(filename);
 					level_type_was = level_type;
 					changed = false;

